Dockerfile

```diff
@@ -21,6 +21,9 @@ RUN export DEBIAN_FRONTEND=noninteractive \
  && apt-get install -y --no-install-recommends \
       apache2 \
       ca-certificates \
+      ca-cacert \
```

Comment:
Please sort the list, see [Dockerfile best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments).